File: 1. Two Sum/twoSum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int[] temp = new int[2];
        for(int i =0; i < nums.length; i++){
            for(int j = 0; j < nums.length; j++){
                if(i == j){
                    continue;
                }else{
                    if(nums[i]+nums[j] == target){
                        temp[0] = i;
                        temp[1] = j;
                        return temp;
                    }
                }
            }
        }
        return temp;
    }
}

File: 1. Two Sum/twoSum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        hashtable = dict()
        # initialize the hashtable
        for i in range(len(nums)):
            if target-nums[i] in hashtable: # search the key of hashtable
            # aka: search the target-nums[i] in the key of hashtable 
            # if found, just return the value of that key
                return [hashtable[target-nums[i]],i]
            hashtable[nums[i]] = i #store the value of nums as indexs of hashtable 
        
        return []
